README.rst

```diff
@@ -95,9 +95,10 @@ This plugin is disabled by default! To enable it::
 Known issues
 ------------
 
-If you reference foreign-key models by their name (as string) ``pylint-django`` may not be
-able to find the model and will report issues because it has no idea what the underlying
-type of this field is. Supported options are::
+If Django is not installed and  you reference foreign-key models by their name (as
+string) ``pylint-django`` may not be able to find the model and will report issues
+because it has no idea what the underlying type of this field is. Supported options
+are::
```

Comment:
Why the change here ? I don't see anything different in the text.

Reply:
I added some additional text to the first line `If Django is not installed...`. But this was before I knew about `checkers/django_installed.py` so I'll undo this change (don't think it makes much sense anymore).

Reply:
I undid my earlier change but maybe the entire known issues section should be remove if this is accepted.